### ISSUE TITLE
Different Eyes Effect

### ISSUE THREAD

**Issue by NorNoga4dm1n** [Aircoookie/WLED#2237]
coming up on the Halloween time, I thought I would start looking into some effects to run for the holiday and came acrost the Halloween Eyes effect. I like it, however, one set of eyes is too limited for my liking and made me think of an improved version...

color1 define eye color
color2 defines backround color
color3 is unused
palette override eye colors (AKA multi eye colors)
speed slider fade rate
intensity slider delay before spawning new eyes

effect starts by drawing the background color on all squares then "fades" all squares to background color
after random timer spawn eye, voiding spawn if overlap

in other words, im looking for more eyes that fade slowly

**Comment by huggy-d1** [comment]
With that effect definition, you should be able to create a new effect and integrate to WLED in no time, submit a pull request to have it integrated into the master version of WLED so everyone can use it.

Looking forward with excitement whatever you come up with.
